src/System.IdentityModel.Tokens.Jwt/JwtSecurityTokenHandler.cs

```diff
@@ -1056,12 +1056,7 @@ protected virtual ClaimsIdentity CreateClaimsIdentity(JwtSecurityToken jwt, stri
             {
                 throw new ArgumentNullException("jwt");
             }
-
-            if (string.IsNullOrWhiteSpace(issuer))
-            {
-                throw new ArgumentException(ErrorMessages.IDX10221);
```

Comment:
We cannot remove it since this error message is used in somewhere else, like SamlSecurityTokenHandler.cs.